homework14/js/main.js

```diff
@@ -56,23 +29,26 @@ const textButton = document.querySelector('.second-block__button');
 const resultNewString = document.querySelector('.second-block__result');
 
 function vowelOne(inputString){
-    let vowels = 'aeiouy';
-    let consonants = 'bcdfgklmnpqstvxzhrw';
-    let symbols = '.,- '
+    const vowels = ['a','e','i','o','u','y'];
+    const consonants = ['b','c','d','f','g','j','k','l','m','n','p','q','s','t','v','x','z','h','r','w'];
+    const symbols = ['.', ',', '-', ' ', '!', '`', '₴', '~', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '=',
+        '+', ';', ':', '?', '<', '>', '|', '/', '{', '}', '[', ']', '№', '"']
+    const numbers = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
```

Comment:
You should not create such an array with incorrect symbols. Just build the program in that way - everything that is not a vowel - is 0. That's simple

Reply:
`consonants`, `symbols` and `numbers` must be removed